app/index.js

```diff
@@ -236,4 +87,4 @@ module.exports = class extends Generator {
       skipInstall: this.options['skip-install']
     });
   }
-};
+};
```

Comment:
Let's turn on final newlines in your text editor 😄 Which one are you using? Generally it should be enough to install an EditorConfig extension, because we have it configured in `.editorconfig`.

Reply:
😓 yeah! I am using c9.io and right now is not supported, but I can fix with eslint... let me check that point also in #741